astropy/modeling/utils.py

```diff
@@ -493,3 +493,190 @@ def combine_labels(left, right):
+            limited to the bounding_box of the model. The the bounding_box
+            is not defined, an image or coordinates must be passed.
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        import matplotlib.pyplot as plt
+        from astropy.modeling.models import Ellipse2D
+        from astropy.modeling.utils import ellipse_extent
+
+        amplitude = 1
+        x0 = 50
+        y0 = 50
+        a = 30
+        b = 10
+        theta = np.pi/4
+
+        model = Ellipse2D(amplitude, x0, y0, a, b, theta)
+
+
+        dx, dy = np.around( ellipse_extent(a, b, theta) )
+
+        limits = [slice(y0 - dy, y0 + dy + 1, 1),
+                  slice(x0 - dx, x0 + dx + 1, 1)]
+
+        coords = np.mgrid[limits][::-1]
+
+        plt.imshow(model(*coords),cmap='binary',interpolation='nearest',alpha=.5,
+                  extent = [x0 - dx, x0 + dx, y0 - dy, y0 + dy])
+        plt.show()
+    """
+
+    t = np.arctan2(-b * np.tan(theta), a)
+    dx = a * np.cos(t) * np.cos(theta) - b * np.sin(t) * np.sin(theta)
+
+    t = np.arctan2(b, a * np.tan(theta))
+    dy = b * np.sin(t) * np.cos(theta) + a * np.cos(t) * np.sin(theta)
+    return np.abs([dx, dy])
+
+
+def render_model(model, image=None, coords=None, copy=True):
+    """
+    Evaluates a model on an input array. Evaluation is limited to
+    a bounding box if the `bounding_box` property is defined for
+    the model.
+
+    Parameters
+    ----------
+    model : Model
+            Model to be evaluated
+    image : array
+            Array on which the model is evaluated.
+    coords: list
+            List of coordinate arrays mapping to the image array,
+            such that `image[coords] == image`.
+    copy : bool, optional
+           Default is true. If False, image will be modified in place. 
+
+    Returns
+    -------
+    image : array
+            The evaluated model on the input image or coordinates. If 
+            no image or coordinates are input, the returned array is 
+            limited to the bounding_box of the model. The the bounding_box
+            is not defined, an image or coordinates must be passed.
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+            import numpy as np
+        import matplotlib.pyplot as plt
+
+        from astropy.modeling.models import Gaussian2D
+        from astropy.modeling.utils import render_model
+
+        imshape = (450, 600)
+        nsrc = 50
+        models = {}
+        models['x_mean'] = np.random.rand(nsrc) * imshape[1]
+        models['y_mean'] = np.random.rand(nsrc) * imshape[0]
+        models['amplitude'] = np.abs(np.random.randn(nsrc) + 10.)
+        models['x_stddev'] = np.abs(np.random.rand(nsrc) + 5)
+        models['y_stddev'] = models['x_stddev'] * (np.random.rand(nsrc) + 1)
+        models['theta'] = np.random.rand(nsrc) * np.pi
+
+        model_list = [Gaussian2D(x_mean=models['x_mean'][i], y_mean=models['y_mean'][i], 
```

Comment:
I think it could be worth splitting this out into a utility function `make_model_list_from_table`?
@embray Would it be welcome as part of the official `astropy.modeling` API or is it doing too little / not common enough?